nevergrad/parametrization/core.py

```diff
@@ -276,9 +276,17 @@ def satisfies_constraints(self) -> bool:
         if not self._constraint_checkers:
             return True
         val = self.value
-        return all(func(val) for func in self._constraint_checkers)
+        def ok(x: tp.Union[bool, float]) -> bool:
```

Comment:
add a comment below to explain why this is needed. I know why now, but eventually that informationn may get lost.

Reply:
Done.